server/views/map.html

```diff
@@ -33,6 +33,22 @@
 {% block main %}
 {% block content %}
   <main class="flood-map-main" id="main-content">
+    <noscript>
+      <div class="fm-error">
```

Comment:
Note that Dan had a more pithy statement, but I have swapped it for the sentence we display elsewhere, and added "and a supported device", because this will also show on old devices that don't support the map.